src/app/core/export/download-service/download.service.spec.ts

```diff
@@ -99,15 +100,19 @@ describe("DownloadService", () => {
     testEntity.dateProperty = new Date(testDate);
     testEntity.boolProperty = true;
 
+    const objectToArray = [
+      testEntity.enumProperty.label,
+      testEntity.dateProperty.toISOString(),
+      testEntity.boolProperty.toString(),
+      ];
+      
     const csvExport = await service.createCsv([testEntity]);
```

Comment:
`TestEntity`'s fields do not have any labels here, so this is not really testing the new functionality.

Let's update above:
```suggestion
        @DatabaseEntity("TestEntity")
    class TestEntity extends Entity {
      @DatabaseField({ "label": "test enum" }) enumProperty: ConfigurableEnumValue;
      @DatabaseField({ "label": "test date" }) dateProperty: Date;
      @DatabaseField({ "label": "test boolean" }) boolProperty: boolean;
    }
```

Can you also add expectations (or a separate unit test) to check that the header indeed exports the field labels?